libs/api/domains/driving-license-book/src/dto/createPracticalDrivingLesson.input.ts

```diff
@@ -0,0 +1,19 @@
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType()
+export class CreatePracticalDrivingLessonInput {
+  @Field()
+  bookId!: string
+
+  @Field()
+  teacherNationalId!: string
```

Comment:
Should driving license teachers be able to create driving licenses for other teachers?

If not, then this input field should not be here, and it should be using the currently logged in user's national id (which will be a driving license teacher)

Reply:
Fixed.